supervisor/supervisor.py

```diff
@@ -244,6 +246,18 @@ async def repair(self):
             _LOGGER.error("Repair of Supervisor failed")
 
     async def check_connectivity(self):
+        """Check the connection."""
+        if self.connectivity:
+            await self._check_connectivity_throttled()
+        else:
+            await self._check_connectivity()
```

Comment:
Let's not do that, we created functions just to avoid adding a job feature. Let's add this as a feature to the job decoration to skip throttling on a variable

Reply:
Allowed someone to pass in a lambda that returns a timedelta instead of a timedelta for `throttle_period`. Thought that makes it pretty clean, let me know what you think.